kiwi_keg/generator.py

```diff
@@ -103,6 +110,32 @@ def create_kiwi_description(self, overwrite: bool = False) -> None:
             kiwi_config.write(kiwi_document)
             kiwi_config.write('\n')
 
+    def create_xml_description(self, overwrite) -> None:
```

Comment:
Not doing anything with `overwrite` argument. Presumably there should be a check

``if not overwrite and os.path.exists(....):
      raise .....
```